<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="KURALGAUTHAM" failures="2" tests="9" name="Test" time="0.403" errors="0" timestamp="25 May 2021 15:41:57 GMT">
  <testcase classname="org.testNG.Sample2" name="tc11" time="0.048"/>
  <testcase classname="org.testNG.Sample2" name="tc22" time="0.032">
    <skipped/>
  </testcase> <!-- tc22 -->
  <testcase classname="org.testNG.Sample2" name="tc22" time="0.012">
    <skipped/>
  </testcase> <!-- tc22 -->
  <testcase classname="org.testNG.Sample2" name="tc22" time="0.005">
    <skipped/>
  </testcase> <!-- tc22 -->
  <testcase classname="org.testNG.Sample2" name="tc22" time="0.042">
    <skipped/>
  </testcase> <!-- tc22 -->
  <testcase classname="org.testNG.Sample2" name="tc22" time="0.016">
    <skipped/>
  </testcase> <!-- tc22 -->
  <testcase classname="org.testNG.Sample2" name="tc22" time="0.02">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at org.testNG.Sample2.tc22(Sample2.java:14)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- tc22 -->
  <testcase classname="org.testNG.Sample2" name="tc33" time="0.013">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at org.testNG.Sample2.tc33(Sample2.java:19)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- tc33 -->
  <testcase classname="org.testNG.Sample2" name="tc44" time="0.007"/>
</testsuite> <!-- Test -->
